src/libraries/System.Diagnostics.Process/tests/ProcessTestBase.cs

```diff
@@ -74,6 +86,20 @@ protected Process CreateProcess(Func<string, int> method, string arg, bool autoD
             return p;
         }
 
+        protected Process CreateProcess(Func<string, Task<int>> method, string arg, bool autoDispose = true)
```

Comment:
It doesn't look like you use this autoDispose; you can just remove it and always call AddProcessForDispose.

Reply:
Since accepting the code review feedback in this thread this function isn't used anymore, so I just removed it.